test/test_tensor_creation_ops.py

```diff
@@ -1145,6 +1145,12 @@ def test_full_out(self, device):
         self.assertEqual(torch.full(o.shape, 1., out=o).dtype, o.dtype)
         self.assertEqual(torch.full(size, 1, out=o).dtype, o.dtype)
 
+    def test_tensor_from_non_writable_numpy(self, device):
```

Comment:
Add a comment with the issue this test addresses so people can reference it

Reply:
https://github.com/pytorch/pytorch/pull/47271/commits/a4101073298241bacb0d900e7e3a94cf1b4c204a adds the comment

Reply:
Did the force push remove the comment including the issue?

Reply:
No

Reply:
Sorry I'm confused. Where is the comment with the issue?

Reply:
https://github.com/pytorch/pytorch/commit/44ff10ed7ae884eddb74a9472620b3b8e3c92a62 This contains the comment

Reply:
@ArtistBanda I think @mruberry wants you to add a reference to the issue in this comment. Either the issue ID or the URL to the issue (https://github.com/pytorch/pytorch/issues/47160).

Reply:
That's right, thank you @thomasjo. I was expected a comment like

```
# Tests for X, see (link to issue)
```

Reply:
Oh okay, sorry I didn't get that before. I'll add it now.

Reply:
https://github.com/pytorch/pytorch/pull/47271/commits/187a07e1faf2d8f08f49d9b4107a8e69c7ccef1d resolves this